### ISSUE TITLE
[DISCOVERY] Investigate if we can export REACT bindings and do we need to 

### ISSUE THREAD

**Issue by caw310** [department-of-veterans-affairs/va.gov-team#26873]
Do discover of whether we can  export REACT bindings and do we need to and identify a path forward.

**Comment by cvalarida** [comment]
**TL;DR:** We won't spend time to set up Stencil to output the React bindings. This is probably useful eventually to keep the usage less "magic" in React—and to pass non-scalar data such as objects and arrays—but for now, the benefits don't justify the cost.

We can revisit this decision later, though.

---

After some experimentation, we discovered the issues I ran into aren't a problem in `vets-website`. Specifically:
- We're able to pass boolean values to a web component and it works fine, even though the DOM says it's `checked="false"` which is technically incorrect according to the WHATWG specification
- We're able to use `onEventName` props to attach event handlers
  - This wasn't working in my barebones experimentation project, but it _is_ working in `vets-website`
  - So this may come up again later for folks using the component library outside of VA.gov, but that's a problem we can solve later
    - @bkjohnson and I couldn't figure out _why_ it wasn't working in my experimentation project, nor what was different between that and `vets-website`
    - At this point, I don't believe it's worth our time to continue troubleshooting it
